packages/cli/src/commands/build/index.ts

```diff
@@ -426,6 +426,18 @@ async function doBuild(
 
   const ops: Promise<Error | void>[] = [];
 
+  const isUsingSpeedInsights = Boolean(
+    pkg?.dependencies?.['@vercel/speed-insights']
```

Comment:
**thought:** what if user includes it in their `devDependencies`?

It seems that next [makes no distinction](https://github.com/vercel/next.js/issues/43066#issuecomment-1319969015) since every required lib will be bundled together.

As a vite user, I personally puts everything in `devDependencies` because eventually, there are no node modules for a bundle app.